### PR TITLE
Remove test files from release package

### LINKED ISSUE #700: Remove unwanted files from distributed release and/or whole system
Some files may want to remain within Guiguts, but not be distributed. Some might go completely
1. scannos/GutWrenchGutAxe folder?
2. tests folder?
3. tools/bookloupe/*.tst files?

### PR DESCRIPTION
**Pull request by windymilla**
1. Move the GG test directory from under src to the directory above so it does not
get included. Edit the test code to cope with the new location
2. Remove the bookloupe test program and test files when packaging it
3. Fix typo in bookloupe README

Note that the GutWrenchGutAxe scannos were unknown to PPers, but once they
knew they existed, they didn't want to lose them, so this commit leaves them alone.

Fixes #700

### CODE REVIEW

File: tests/epubfriendly.html [hunk 1]
```diff
@@ -1,76 +1,76 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
```
Comment (cpeel) [inline tests/epubfriendly.html:1]: I'm confused on why these files are listed as changed -- apparently by whitespace? Did the line endings change? How did you move the files, `git mv src/tests .`?

**Review by cpeel — COMMENTED** [review]
> Note that the GutWrenchGutAxe scannos were unknown to PPers, but once they knew they existed, they didn't want to lose them, so this commit leaves them alone.

I'm not sure if that's a reason to leave them or a reason to remove them. Such hoarders! 😁

File: tests/epubfriendly.html (hunk 1 above)
Comment (windymilla) [reply tests/epubfriendly.html:1]: No, I just moved the folder in Windows. Was that wrong?

Comment (cpeel) [reply tests/epubfriendly.html:1]: That should have been fine. I'm not too worried about it as viewing the diffs without whitespace shows no changes. Mostly curious.

Comment (windymilla) [reply tests/epubfriendly.html:1]: Thanks

**Pull request merged**